### PR TITLE
[plm-spare] Beautify

### PR DESCRIPTION
**Pull request by SalahAdDin**
Updated to `pep8` and **Odoo** `11`.

### CODE REVIEW

File: plm_spare/models/plm_document_extension.py [hunk 1]
```diff
@@ -20,22 +19,20 @@
...
+from odoo import models
+from odoo import fields
+from odoo import _
 
 
 class PlmDocumentExtension(models.Model):
     _inherit = 'plm.document'
 
-    usedforspare = fields.Boolean(_('Used for Spare'),
+    used_for_spare = fields.Boolean(_('Used for Spare'),
```
Comment (OmniaGit) [inline plm_spare/models/plm_document_extension.py:36]: replace with the old name please...

File: plm_spare/models/plm_temporary.py [hunk 2]
```diff
@@ -19,46 +18,45 @@
...
+_logger = logging.getLogger(__name__)
 
 
-class plm_temporary(osv.osv.osv_memory):
+class PlmTemporary(osv.osv.osv_memory):
     _inherit = "plm.temporary"
 
     @api.multi
-    def action_create_spareBom(self):
+    def action_create_spare_bom(self):
```
Comment (OmniaGit) [inline plm_spare/models/plm_temporary.py:33]: replace with the old name

File: plm_spare/models/product_product_extension.py [hunk 3]
```diff
@@ -20,77 +20,76 @@
...
+from odoo import models
+from odoo import api
 
 
 class PlmComponentExtension(models.Model):
     _inherit = 'product.product'
 
     @api.multi
-    def action_create_spareBom_WF(self):
+    def action_create_spare_bom_wf(self):
```
Comment (OmniaGit) [inline plm_spare/models/product_product_extension.py:37]: replace with the old name

File: plm_spare/report/spare_parts_manual.py [hunk 4]
```diff
@@ -19,106 +18,100 @@
...
-def isPdf(fileName):
-    if (os.path.splitext(fileName)[1].lower() == '.pdf'):
+def is_pdf(file_name):
+    if os.path.splitext(file_name)[1].lower() == '.pdf':
         return True
     return False
 
 
-def getDocumentStream(docRepository, objDoc):
+def get_document_stream(doc_repository, obj_doc):
```
Comment (OmniaGit) [inline plm_spare/report/spare_parts_manual.py:50]: replace with old name

File: plm_spare/report/spare_parts_manual.py [hunk 5]
```diff
@@ -19,106 +18,100 @@
...
     return content
 
 
 def _translate(value):
     return _(value)
 
 
-def BomSort(myObject):
-    bomobject = []
+def bom_sort(my_object):
```
Comment (OmniaGit) [inline plm_spare/report/spare_parts_manual.py:70]: replace with old name

File: plm_spare/report/spare_parts_manual.py [hunk 6]
```diff
@@ -19,106 +18,100 @@
...
-            myObject.product_qty,
+            my_object.product_qty,
             '',
-            myObject.weight_net,
+            my_object.weight_net,
             ]
 
 
-class report_spare_parts_header(models.AbstractModel):
+class ReportSparePartsHeader(models.AbstractModel):
```
Comment (OmniaGit) [inline plm_spare/report/spare_parts_manual.py:98]: replace with old name

File: plm_spare/report/spare_parts_manual.py [hunk 7]
```diff
@@ -135,81 +128,81 @@ def create(self, components):
...
+            return byte_string.decode('UTF-8')
         logging.warning('Unable to create PDF')
-        return (False, '')
+        return False, ''
 
-    def getSparePartsPdfFile(self, product, output, componentTemplate, bomTemplate, recursion):
-        packedObjs = []
-        packedIds = []
-        if product in self.processedObjs:
+    def get_spare_parts_pdf_file(self, product, output, component_template, bom_template, recursion):
```
Comment (OmniaGit) [inline plm_spare/report/spare_parts_manual.py:153]: replace with old name

File: plm_spare/report/spare_parts_manual.py [hunk 8]
```diff
@@ -135,81 +128,81 @@ def create(self, components):
...
                     if recursion:
-                        for packedObj in packedObjs:
-                            if packedObj not in self.processedObjs:
-                                self.getSparePartsPdfFile(packedObj, output, componentTemplate, bomTemplate, recursion)
+                        for packed_obj in packed_objs:
+                            if packed_obj not in self.processed_objs:
+                                self.get_spare_parts_pdf_file(packed_obj, output, component_template, bom_template, recursion)
 
-    def getPdfComponentLayout(self, component):
+    def get_pdf_component_layout(self, component):
```
Comment (OmniaGit) [inline plm_spare/report/spare_parts_manual.py:183]: replace with old name

File: plm_spare/report/spare_parts_manual.py [hunk 9]
```diff
@@ -135,81 +128,81 @@ def create(self, components):
...
-                    value = getDocumentStream(docRepository, document)
+                elif is_pdf(document.datas_fname):
+                    value = get_document_stream(doc_repository, document)
                     if value:
                         ret.append(BytesIO(value))
         return ret
 
-    def getFirstPage(self, ids):
-        strbuffer = BytesIO()
+    def get_first_page(self, ids):
```
Comment (OmniaGit) [inline plm_spare/report/spare_parts_manual.py:196]: replace with old name

File: plm_spare/report/spare_parts_manual.py [hunk 10]
```diff
@@ -135,81 +128,81 @@ def create(self, components):
...
         pdf = self.env.ref('plm_spare.report_product_product_spare_header').sudo().render_qweb_pdf(ids)[0]
-        strbuffer.write(pdf)
-        return strbuffer
+        str_buffer.write(pdf)
+        return str_buffer
 
     @api.model
-    def get_report_values(self, docids, data=None):
-        documents = self.env['product.product'].browse(docids)
+    def get_report_values(self, doc_ids):
```
Comment (OmniaGit) [inline plm_spare/report/spare_parts_manual.py:204]: replace with old name

**Review by OmniaGit — CHANGES_REQUESTED** [review]
please leave the function name as it was.. it will possible brake the client application due to xml-rpc calls

**Comment by SalahAdDin** [comment]
@OmniaGit But, the old name is not in snake case which is a `pep8` error.

**Review by OmniaGit — APPROVED** [review]

**Pull request merged**